translate/storage/jsonl10n.py

```diff
@@ -416,6 +416,128 @@ def _extract_units(
             yield from parent
 
 
+class GoTextJsonUnit(BaseJsonUnit):
+    ID_FORMAT = "{}"
+
+    def __init__(
+        self,
+        comment=None,
+        message=None,
+        meaning=None,
+        key=None,
+        fuzzy=None,
+        position=None,
+        **kwargs,
```

Comment:
Please keep existing positional args before new ones, otherwise something will break as that changes API.

Reply:
I don't know what this means, sorry, none of these new arguments are positional, they're all named?

Reply:
Nevermind, I found an example and I see what you mean. Done.